eslint.config.js

```diff
@@ -31,6 +31,11 @@ export default tseslint.config(
 	jsonc.configs["flat/recommended-with-json"],
 	markdown.configs.recommended,
 	n.configs["flat/recommended"],
+	{
+		rules: {
+			"n/no-missing-import": "off",
+		},
```

Comment:
Let's move this down to the first `rules: { ... }` block later down, just before the `// Stylistic concerns ...` comment. No need for a separate block for one rule.